install/ci-vm/ci-linux/startup-script.sh

```diff
@@ -21,10 +21,10 @@ gcs_bucket=$(curl http://metadata/computeMetadata/v1/instance/attributes/bucket
 vm_name=$(curl http://metadata.google.internal/computeMetadata/v1/instance/hostname -H "Metadata-Flavor: Google")
 vm_name=(${vm_name//./ })
 
-echo "${gcs_bucket} /repository/temp       gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=TestData/ci-linux  0 0" | sudo tee -a /etc/fstab
-echo "${gcs_bucket}   /repository/vm_data         gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=vm_data/${vm_name}  0 0" | sudo tee -a /etc/fstab
-echo "${gcs_bucket}   /repository/TestFiles     gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=TestFiles  0 0" | sudo tee -a /etc/fstab
-echo "${gcs_bucket}   /repository/TestResults     gcsfuse rw,uid=33,gid=33,noatime,async,_netdev,noexec,user,implicit_dirs,allow_other,only_dir=TestResults  0 0" | sudo tee -a /etc/fstab
```

Comment:
In this change, we are removing the ownership to make sure that the repository is accessible by ALL other users of the system, right? Is there a way by which we can make only uid=33 and ftpuser the owners of the repository?

Reply:
I think keeping the `gid` setting would solve the issue.

Reply:
Actually, No...
This script is run inside Linux test VMs, and I realized that there is no importance of user/group `www-data` in VMs; all startup scripts are run as `sudo` user as per the [official documentation](https://cloud.google.com/compute/docs/instances/startup-scripts/linux#:~:text=Runs%20the%20startup%20script%20as%20the%20root%20user%20when%20the%20VM%20boots). Hence `uid=33,gid=33` was redundant. Therefore I removed it.

This has no relation with bucket permissions, in that `gid` is set, so only users of that group have the bucket permissions...

> In this change, we are removing the ownership to make sure that the repository is accessible by ALL other users of the system, right? Is there a way by which we can make only uid=33 and ftpuser the owners of the repository?

Reply:
Oh... I see. Thanks for taking the time to help me understand.